src/components/EntityInfo/EntityMenu.vue

```diff
@@ -285,6 +328,14 @@ export default class EntityMenu extends Mixins(AllowableActionsMixin) {
     }
   }
 
+  get amalgamateTooltipText (): string {
+    if (this.isAdminFrozen) {
+      return 'This business is frozen and cannot be involved in an amalgamation.'
+    } else {
+      return 'This business has a future effective dissolution and cannot be involved in an amalgamation.'
+    }
```

Comment:
I recommend an explicit check for `isPendingDissolution` instead of assuming it as the fallback case.

As a fallback, you could return null, and then you could display the tooltip only if `amalgamateTooltipText` is not falsy.

Reply:
Fixed, thanks.

Reply:
Fixed, thanks.